pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -511,5 +511,6 @@ message LogicalReplicationWriterSpec {
     // Checkpoint stores a set of resolved spans denoting completed progress.
     optional jobs.jobspb.StreamIngestionCheckpoint checkpoint = 7 [(gogoproto.nullable) = false];
 
-    map<string, cockroach.sql.sqlbase.TableDescriptor> table_descriptors = 8 [(gogoproto.nullable) = false];
+    // TableDescriptors is a map between the destination table IDs and the source table descriptor.
```

Comment:
nit: comment could be reworded. "map from destination table ID to  source table descriptor"